plugins/kubernetes-backend/api-report.md

```diff
@@ -298,6 +298,7 @@ export interface KubernetesFetcher {
   fetchPodMetricsByNamespaces(
     clusterDetails: ClusterDetails,
     namespaces: Set<string>,
+    labelSelector?: string,
```

Comment:
This was added by build:api-reports. I was hesitant to add it to the PR, but the GitHub action said that I needed to add all updates from it.

Reply:
This is in main now https://github.com/backstage/backstage/blob/master/plugins/kubernetes-backend/api-report.md?plain=1#L301 so if you rebase main it will probably go away